lewis_emulators/tpg300/device.py

```diff
@@ -55,6 +55,25 @@ class ReadState(Enum):
     FSB = "FSB"
     SPS = "SPS"
 
+    """ 
+    This object represents settings for a circuit in the device.
+        these settings are: high_threshold(float), high_exponent(int),
+        low_threshold(float), low_exponent(int), circuit_assignment(1|2|2|4|A|B)
+    """
+class CircuitAssignment:
+    high_threshold = 0.0
+    high_exponent = 0
+    low_threshold = 0.0
+    low_exponent = 0
+    circuit_assignment = 1
```

Comment:
It looks like these *class variables* aren't used; we're only using the *instance variables* defined in `__init__` below? I think these lines can probably be removed.